<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Cytoscape Plugin for MiMI</title>
<link href="mimipluginstyle.css" rel="stylesheet" type="text/css" />
</head>

<body>
<div id="wrapper">
<div id="header"><img src="images/MiMI_Plugin_banner.jpg" alt="MiMI Plugin Banner" width="1000" height="60" /></div>
<div id="navcontainer">
<ul id="navlist">
<li><a href="index.html" >Home</a></li>
<li><a href="tryplugin.html">Try Plugin</a></li>
<li><a href="tutorial.html">Tutorials</a></li>
<li><a href="download.html">Download</a></li>
<li id="active"><a href="acknowledgement.html" id="current">Acknowledgement</a></li>
<li ><a href="bugs.html" >Known Bugs</a></li>
<li><a href="contact.html">Contact</a></li>
</ul>
</div>
<div id="content">
  <h3>Citing MiMI</h3>
  <hr />
  <p>Our <strong>Cytoscape MiMI Plugin</strong> queries interaction from MiMI and visualizes returned network with Cytoscape. Our plugin also integrates BioNLP database for literature information, MEAD for document summarization and SAGA for pathway matching. Here, we want to thank Cytoscape for providing this fabulous tool. We also thank <a href="http://ccmb.med.umich.edu" target="_blank">CCMB</a> and <a href="http://www.ncibi.org">NCIBI</a> at <a href="http://www.umich.edu" target="_blank">University of Michigan</a> for providing MiMI, BioNLP, MEAD and SAGA. <br />
  </p>
  <p><a href="http://www.cytoscape.org/" target="_blank">Cytoscape</a><br />
    Cytoscape is an open source bioinformatics software platform for visualizing molecular interaction networks and integrating these interactions with gene expression profiles and other state data. </p>
  <p><a href="http://mimi.ncibi.org/MiMI/home.jsp" target="_blank">MiMI</a><br />
    Michigan Molecular Interactions (MiMI) assists scientists digging through an overwhelming amount of protein interaction data. MiMI gathers data from well-known protein interaction databases and merges the information. Since these datasets often contain contradictory information, the merge is performed in such a way that all information is retained. A provenance model has been developed that tracks where each piece of data came from, and what processes have been performed upon it. A simple yet powerful user interface aids users in their queries. Thus, MiMI allows scientists to query all data, whether corroborative, or contradictory, and specify which sources to utilize</p>
  <p><a href="http://enigma.eecs.umich.edu/saga.html">SAGA</a><br />
    SAGA stands for Subgraph Index for Approximate Graph Alignment. It is an efficient tool for approximate subgraph matching. SAGA allows users to match a query graph against a large database of graphs. At the core of SAGA is a flexible graph distance model that incorporates node approximate matching as well as approximate structure matching. A powerful indexing method is implemented to speed up the matching process. Some applications of SAGA include querying/comparing pathways and querying parsed biomedical literature databases to find similar documents. </p>
  <p><a href="http://www.summarization.com/mead/" target="_blank">MEAD</a><br />
    MEAD is a public domain portable multi-document summarization system, developed by University of Michigan Information School. MEAD implements multiple summarization algorithms (at arbitrary compression rates) such as position-based, Centroid [RJB00], TF*IDF, and query-based methods. Methods for evaluating the quality of the summaries include co-selection (precision/recall, kappa, and relative utility) and content-based measures (cosine, word overlap, bigram overlap).</p>
  <p><a href="https://portal.ncibi.org/portal/site/!gateway/page/9b2c0673-4252-409d-003d-25996426b215" target="_blank">BioNLP</a><br />
    SQL database with all PubMed baseline and update records, sentence-segmented and named entity tagged for genes, proteins, and MeSH terms. It contains over 17 million citations from Medline and other life sciences journals. Updated nightly</p>
  <p></p>
  <h3>Credits</h3>
  <hr />
  <p>MiMI and the MiMI Cytoscape Plugin are supported by generous grants from:<br />
      <br />
    • <a href="http://www.nsf.gov/" target="_blank">National Science Foundation: Grant IIS 0219513 </a><br />
    • <a href="http://www.nih.gov/">National Institutes of Health: Grants R01 LM008106 and U54 DA021519</a> <br />
    • <a href="http://www.ncibi.org/" target="_blank">National Center for Integrative Biomedical Informatics</a> <br />
    • <a href="http://www.ctaalliance.org/MCBI/index.html" target="_blank">Michigan Center for Biological Information</a> <br />
    • <a href="http://www.promisingminds.com/" target="_blank">Pfizer, Medical &amp; Academic Partnerships (MAP) </a><br />
    • <a href="http://www.hhmi.org/" target="_blank">Howard Hughes Medical Institute (HHMI)</a> <br />
    • <a href="http://www.microsoft.com/" target="_blank">Microsoft Corporation </a></p>
  <h3>&nbsp;</h3>
  <br />
</div>
<div id ="footer">
  <div align="center">
    <p><br />
      </p>
  </div>
</div>
</div>
</body>
</html>
